app/src/components/ApproveModal.tsx

```diff
@@ -18,7 +18,7 @@ const ApproveModal: React.FC<ApproveModalProps> = ({ isOpen, onClose, token, use
 
   const handleAccept = async () => {
     try {
-      await AuthServices.AcceptApplication(applicationId);
+      await AuthServices.AcceptApplication(applicationId);    // <- applicationId is actually baseUser!
```

Comment:
Is it not easier to change the prop name rather than add a comment future users have to adhere to?

Reply:
To be fair, we approve a content creator based on its application so I think applicationId is fine. But if we decide to rename it, also do it for RejectModal. Thus, remember to also rename the props that get passed in **UserDetailsModal** on line 207-208.

Reply:
@LazarShockX 
 > To be fair, we approve a content creator based on its application so I think applicationId is fine. But if we decide to rename it, also do it for RejectModal. Thus, remember to also rename the props that get passed in **UserDetailsModal** on line 207-208.

I _strongly_ contemplated doing that, but ended up only renaming it for **ApproveModal.tsx** and the single place it's invoked in **UserDetailsModal.tsx**. The reference is passed around in five different files and multiple functions, so I didn't dare it as I was afraid I might wreck something somewhere. Also, I just don't have the time or energy to test the affected functionalities (which is why I didn't want to rename it to begin with 😬).

Reply:
All good, nice job 🩵